Assets/Mirror/Examples/Lobbey/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,120 @@
+using UnityEngine;
+
+namespace Mirror.Examples.NetworkLobby
+{
+    [RequireComponent(typeof(CharacterController))]
+    public class PlayerController : NetworkBehaviour
+    {
+        [SyncVar]
+        public int Index;
+
+        [SyncVar]
+        public uint score = 0;
+
+        [SyncVar(hook = nameof(SetColor))]
+        public Color playerColor = Color.black;
+
+        CharacterController characterController;
+
+        public float moveSpeed = 300f;
```

Comment:
'f' suffix not needed. you don't use it for 'maxTurnSpeed' etc. either

Reply:
Added the missing `f` in the others for explicit assignment instead of implicit conversion.